code/modules/antagonists/cult/cult_items.dm

```diff
@@ -190,6 +194,107 @@ Striking a noncultist, however, will tear their flesh."}
 			SWORD_PREFIX = "stillborn", // lol loser
 		) ,
 	)
+	actions_types = list(/datum/action/item_action/haunted_blade)
+
+/obj/item/melee/cultblade/haunted/examine(mob/user)
+	. = ..()
+
+	var/examine_text = ""
+	if(bound)
+		examine_text = "[src] shines a dull, sickly green, the power emanating from it clearly bound by the runes on its blade. You could unbind it, and wield its fearsome power. But is it worth loosening the bindings of the spirit inside?"
+	else
+		examine_text = "[src] flares a bright and malicious pale lime shade. Someone has unbound the spirit within, and power now clearly resonates from inside the blade, barely restrained and brimming with fury. You may attempt to bind it once more, sealing the horror, or try to harness its strength as a blade."
+
+	. += span_cult(examine_text)
+
+/datum/action/item_action/haunted_blade
+	name = "Unseal Spirit" // img is of a chained shade
+	button_icon = 'icons/mob/actions/actions_cult.dmi'
+	button_icon_state = "shade_sealed"
+
+/datum/action/item_action/haunted_blade/apply_button_icon(atom/movable/screen/movable/action_button/button, force)
+	var/obj/item/melee/cultblade/haunted/blade = target
+	if(istype(blade))
+		button_icon_state = "shade_[blade.bound ? "sealed" : "unsealed"]"
+		name = "[blade.bound ? "Unseal" : "Seal"] Spirit"
+
+	return ..()
+
+/obj/item/melee/cultblade/haunted/ui_action_click(mob/living/user, actiontype)
+	if(DOING_INTERACTION_WITH_TARGET(user, src))
+		return // gtfo
+	// if(IS_NORMIE())
```

Comment:
What is this line for? 

Are normal crew members supposed to be able to bind / unbind it? Or only magically inclined ones?

Reply:
(IMO, normies shouldn't be able to un/bind it. But you should also update the examine message so it doesn't show up for normies as well)

Reply:
i think it'd be pretty lame if 90% of the crew were unable to interact with the feature

Reply:
besides the trope of unsealing an ancient horror works better if any dumb fool (assistant) can be sweet talked into it